### PR TITLE
UI enhancements

### PR DESCRIPTION
**Pull request by kingofpayne**

### CODE REVIEW

File: laserstudio/instruments/laser.py [hunk 1]
```diff
@@ -9,6 +9,7 @@ def __init__(self, config: dict):
         # Sweep parameters, in order to change the current_percentage
         # regularly, within a random value from sweep_min to sweep_max,
         # each sweep_freq applications
+        self.label = config.get("label")
```
Comment (mmouchous-ledger) [inline laserstudio/instruments/laser.py:12]: This field should be added in the configuration schemas

**Review by mmouchous-ledger — COMMENTED** [review]
Ok for changes, some extra things to do:
- make the cursor position to have fixed size
- add "label" in configuration scheme

**Review by mmouchous-ledger — COMMENTED** [review]
I made the changes and additional UI Enhancement, could you please confirm before explicit approval?
- Viewer's mode (PIN / MOVE / ZONE) selection is unset when button is pressed again (should we also do the same behavior with shortcuts?)
- config.yaml file has the "label" field is attached to any "instrument" object (probe, camera, laser, ...)
- Zoom toolbars's label indicating cursor position is monospaced and have fixed width

**Comment by kingofpayne** [comment]
`should we also do the same behavior with shortcuts?`
I would say no, otherwise the user may disable using the shortcut while believing it is enabling it.

**Review by mmouchous-ledger — APPROVED** [review]

**Pull request merged**